README.md

```diff
@@ -5,11 +5,91 @@
 
 # Utility Library for Impuestos Bolivia
 
+# Usage
+
+To use this library install it from npm:
+
+```
+npm i @nearshorecode/impuestos-bo
+```
+
+or
+
+```
+yarn add @nearshorecode/impuestos-bo
+```
+
+Then you can use it to generate V7 Control code like:
```

Comment:
"like" is too informal and in this sentence is being used as a filler word, could be change to "code, for example:"